foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/OrphedomosToFabric8Migration.java

```diff
@@ -0,0 +1,277 @@
+import com.boozallen.aissemble.upgrade.util.pom.LocationAwareMavenReader;
+import org.apache.maven.model.Build;
+import org.technologybrewery.baton.util.FileUtils;
+import com.boozallen.aissemble.upgrade.util.pom.PomHelper;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.model.InputLocation;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.Plugin;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import com.boozallen.aissemble.upgrade.util.pom.PomModifications;
+import org.codehaus.plexus.util.xml.Xpp3Dom;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.technologybrewery.baton.BatonException;
+
+import static com.boozallen.aissemble.upgrade.util.pom.PomHelper.getLocationAnnotatedModel;
+import static com.boozallen.aissemble.upgrade.util.pom.PomHelper.writeModifications;
+
+import static org.technologybrewery.baton.util.FileUtils.getRegExCaptureGroups;
+
+/**
+ * This migration is responsible for migrating pom files using Orphedomos to use the Fabric8 Docker Maven plugin.
+ */
+
+public class OrphedomosToFabric8Migration extends AbstractAissembleMigration{
+    private static final Logger logger = LoggerFactory.getLogger(OrphedomosToFabric8Migration.class);
+    private static final String OLD_GROUP_ID = "org.technologybrewery.orphedomos";
+    private static final String OLD_ARTIFACT_ID = "orphedomos-maven-plugin";
+    private static final String NEW_GROUP_ID = "io.fabric8";
+    private static final String NEW_ARTIFACT_ID = "docker-maven-plugin";
+    private static String IMAGE_NAME;
+    private static String IMAGE_VERSION;
+    private static final String extractPackagingRegex = "(<packaging>)orphedomos(<\\/packaging>)";
+    private static final String SECOND_REGEX_GROUPING = "$2";
+
+    /**
+     * Function that returns instance of Plugin class with name equal to the pluginName provided
+     * @param model the model to check
+     * @param pluginName the name of the plugin
+     * @return Plugin with the plugin name specified
+     */
+    private Plugin getPlugin(Model model, String pluginName) {
+        Build build = model.getBuild();
+        if (build != null) {
+            List<Plugin> plugins = build.getPlugins();
+            for (Plugin plugin : plugins) {
+                if ((pluginName).equalsIgnoreCase(plugin.getArtifactId())) {
+                    return plugin;
+                }
+            }
+        }
+        return null;
+    }
+
+    private boolean isRootProjectPom(Model model) {
+        return "com.boozallen.aissemble".equals(model.getParent().getGroupId()) && "build-parent".equals(model.getParent().getArtifactId());
+    }
+    private boolean isTestDockerPom(Model model){
+        return model.getParent().getArtifactId().endsWith("-tests") && model.getArtifactId().endsWith("-tests-docker");
```

Comment:
lets bring these string references as final string variables, like you have for the other ones to stay consistent

Reply:
created new final strings:

```
    private static final String ROOT_ARTIFACT_ID = "build-parent";
    private static final String ROOT_GROUP_ID = "com.boozallen.aissemble";
    private static final String TESTS_DOCKER_PARENT_ARTIFACT_ID = "-tests";
    private static final String TESTS_DOCKER_ARTIFACT_ID = "-tests-docker";
```